pkg/healthcheck/grpc.go

```diff
@@ -9,45 +9,42 @@ import (
 	"google.golang.org/grpc"
 )
 
+const grpcApiSubsystemName = "linkerd-api"
```

Comment:
Good question. This subsystem name was originally passed into this package as a variable from the check command in cli/cmd package. It's sort of unclear why that needed to be a variable, however, since it was only ever instantiated once. So I've moved it to a package-private const in this package instead.